policy.md

```diff
@@ -0,0 +1,47 @@
+# Policy
+
```

Comment:
I would recommend an overview of the role policy plays in regulating government software and creating (or not) an enabling environment for leveraging DPGs

Reply:
Thanks.  I've moved this comment into an inline TBD (see [commit 36e40e95](https://github.com/unicef/publicgoods-toolkit/pull/24/commits/36e40e9544f361967b7ff8c5f655aa19111b0472), in preparation for merging the entire in-progress Policy module branch into main, which in turn is in preparation for generating a PDF.